database/seed.js

```diff
@@ -17,14 +17,22 @@ const getDescriptors = () => { // grab vocab words from vocab and capitalize the
   return `${first} ${second}`;
 };
 
+const generatePrice = () => {
+  return `${Math.floor(Math.random() * 200) + 20}.00`; // generates a random price in the range of $20 - $250
```

Comment:
I'm curious, why are you adding 20 to your randomized number?

Reply:
So that the minimum price is $20. My comment about $250 is wrong actually, it should read $220.

Reply:
I see! LGTM!